charts/elasticsearch/templates/nginx/nginx-es-configmap.yaml

```diff
@@ -57,5 +69,8 @@ data:
           proxy_set_header Content-Length "";
           proxy_set_header X-Original-URI $request_uri;
         }
+        location ~ ^/ {
+          deny all;
```

Comment:
Should we add a comment (above somewhere, maybe not right here) stating that any additional proxy passes must be scoped to the deployment index only? Might still be missed, but at least its better than nothing?